splat-1897 Add to contribution guidelines (#1858)
Workiva/frugal: CONTRIBUTING.md

Contributing to Frugal
=======================

## Before you start

> Opening a PR doesn't guarantee success of future merging, it's in your best interest to message the repo owners ahead of time to discuss a feature request. All feature will be evaluated based on their usefulness within the community of user.

Reviewer: ```suggestion
Opening a PR doesn't guarantee a feature will be merged. Please discuss with the repo owners ahead of time to discuss a feature request. All features will be evaluated based on their usefulness within the community of users.
```
